packages/design-system/src/scalars/components/amount-field/amount-field.stories.tsx

```diff
@@ -320,6 +332,17 @@ export const WithError: Story = {
 };
 export const WithMultipleErrors: Story = {
   args: {
+    customValidators: (value: unknown) => {
+      if (
+        typeof value === "object" &&
+        value !== null &&
+        "amount" in value &&
+        value.amount !== "0"
+      ) {
+        return "Tiene que ser igual a cero";
```

Comment:
Please, translate this message to english

Reply:
done